common/scripts/mesh-helper.sh

```diff
@@ -43,21 +43,34 @@ EOF
   fi
 }
 
-generate_br_lan_ip() {
+generate_bridge_ip() {
   local mesh_if_mac
-  mesh_if_mac="$(ip -brief link | grep "$MESH_VIF" | awk '{print $3; exit}')"
+
+  bridge_name=$(echo "$BRIDGE" | cut -d' ' -f1)
```

Comment:
If this function is not for br-lan only but for any "bridge" then please pass the interface name as argument to the function, like so for example: `BRIDGE_NAME="$1"` :)

Reply:
It is not a command line argument.  Configuration sourced to file.

Reply:
fixed in b67d5f9e3c84fcec3732cb04a35297e5acbe08f4

Reply:
Was maybe another block supposed to be marked as resolved? I do see several changes (thank you) in the mentioned commit but I can't seem to find a change related to this? 🤔